client/src/App.js

```diff
@@ -11,46 +11,35 @@ import ProfileSettings from "./pages/ProfileSettings";
 import ProfileDetails from "./pages/ProfileDetails";
 import ProfileListings from "./pages/ProfileListings";
 import BecomeASitter from "./pages/BecomeASitter";
-import PaymentSuccess from "./pages/PaymentSuccess";
 import { AuthProvider } from "./contexts/user";
 import PrivateRoute from "./components/PrivateRoute/";
 import ProfileProvider from "./contexts/profile";
 import MySitters from "./pages/MySitters";
 import MyJobs from "./pages/MyJobs";
-import { STRIPE_PUBLIC_KEY } from "./data/stripe";
-
-const stripe = loadStripe(STRIPE_PUBLIC_KEY);
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Elements stripe={stripe}>
-          <AuthProvider>
-            <Route exact path="/signup" component={SignUp} />
-            <Route path="/login" component={Login} />
-            <ProfileProvider>
-              <Navbar />
-              <Route exact path="/" />
-              <PrivateRoute path="/me" component={ProfileSettings} />
-              <Route exact path="/profiles/" component={ProfileListings} />
-              <Route path="/profiles/:id" component={ProfileDetails} />
-              <PrivateRoute
-                exact
-                path="/payment/success"
-                component={PaymentSuccess}
-              />
-              <PrivateRoute
-                exact
-                path="/become-a-sitter"
-                component={BecomeASitter}
-              />
-              <PrivateRoute exact path="/my-sitters" component={MySitters} />
-              <PrivateRoute exact path="/my-jobs" component={MyJobs} />
-              <PrivateRoute exact path="/messages" />
-            </ProfileProvider>
-          </AuthProvider>
-        </Elements>
+        <AuthProvider>
```

Comment:
Does <Elements> need to be removed for functionality? I'm under the impression that it's needed for payments to work.
@Braagaa your thoughts?

Reply:
Yeah I wasn't sure how to resolve this because those components weren't defined. Michael can you checkout the branch and add this if it's needed?